src/components/entry/base-next-entry.tsx

```diff
@@ -67,32 +67,35 @@ const BaseNextEntry = withText({
   };
 
   return (
-    <div
-      className={`${styles.entry} ${styles.nextEntry} ${sizeClass} ${alwaysShowButtons || (countdown > 0 && showButtons) ? '' : styles.clickable}`}
-      style={{width, color: KalturaPlayer.ui.style.white}}
+    <a
+      tabIndex={0}
+      className={`${styles.entry} ${styles.nextEntry} ${sizeClass}`}
+      style={{width, color: KalturaPlayer.ui.style.white, 'line-height': 'normal'}}
```

Comment:
maybe to move `'line-height': 'normal'` to `a` declaration in css?

Reply:
i've used inline to override the player ui which sets line height to 18px